packages/buckets/src/buckets.ts

```diff
@@ -6,7 +6,8 @@ import { EventIterator } from 'event-iterator'
 import { grpc } from '@improbable-eng/grpc-web'
 import { ContextInterface, Context, defaultHost } from '@textile/context'
 import { UserAuth, KeyInfo } from '@textile/security'
-import { Client } from '@textile/threads-client'
+import { ThreadID } from '@textile/threads-id'
+import { Client } from '@textile/hub-threads-client'
```

Comment:
i needed to move this definition so as to ensure no circular dependencies. but now i'm realizing i could probably just import from threads-client instead of the hub threads. it could still be a good move just to isolate this stuff anyway, let me know what you think

Reply:
Assuming in buckets we don't need to augmented APIs that come with the User module, then yes, it'd be better to just import from threads-client directly. I'd like to keep things mostly contained, plus the more libs on npm there are, the more chances for someone to pull in the wrong thing.

Reply:
So I do use the augmented API, see line 125. So I assume I do need it here.